### PR TITLE
Card interactions

### PR DESCRIPTION
**Pull request by KevinKra**
Include Redux interaction and determine mock data shape for card creation. 

Establish base card styling with unique data passed into all.

### CODE REVIEW

**Comment by vjt960** [comment]
This is really good as an initial setup, but we might end up having to make changes to having separate components based on whether a card is an action, treasure, or victory card. Hopefully not, but is there a reason we're keeping these as separate components as opposed to just having a Card component that will render according to the card data?

**Comment by KevinKra** [comment]
> This is really good as an initial setup, but we might end up having to make changes to having separate components based on whether a card is an action, treasure, or victory card. Hopefully not, but is there a reason we're keeping these as separate components as opposed to just having a Card component that will render according to the card data?

We may need to modify it, the main goal was to get something out and handling an, until that time, undetermined shape of mock data. Additionally, there may be behaviors that could be better encapsulated in a respective component for the card type rather than all populating the state and methods on one all-encompassing card component that renders conditionally based on the passed in data. At the moment, each 'parent' component (treasure, active, victory cards) handles the interpretation of data coming in and then forwards the data to the card to be conditionally rendered there. This approach may end up being more favorable. Though, this will need to be determined as we start figuring out how we handle the different phases going forward.

**Comment by vjt960** [comment]
> 
> We may need to modify it, the main goal was to get something out and handling an, until that time, undetermined shape of mock data. Additionally, there may be behaviors that could be better encapsulated in a respective component for the card type rather than all populating the state and methods on one all-encompassing card component that renders conditionally based on the passed in data. At the moment, each 'parent' component (treasure, active, victory cards) handles the interpretation of data coming in and then forwards the data to the card to be conditionally rendered there. This approach may end up being more favorable. Though, this will need to be determined as we start figuring out how we handle the different phases going forward.

Ah ok. That makes sense, but now I'm trying to see how that works as we implement web-sockets and which aspects of the gameplay the back end is going to handle and what the event objects are going to look like.

File: src/mockData/mockCards.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  },
+  {
+    name: "Province",
+    type: ["Victory"],
+    cost: 8,
+    image: "Province.jpg",
+    victoryPoints: 6,
+    id: 3781
+  }
+];
```
Comment (BrennanAyers) [inline src/mockData/mockCards.js]: 👍

**Pull request merged**